### DISCUSSION TITLE
Who is DimaTorzok?

### DISCUSSION THREAD

**Discussion by NelepovDmitry** [openai/whisper#2372] [Q&A]
Sometimes when I do transcription of audio, I encounter errors in the text -> "Субтитры создавал DimaTorzok"

Why? Who is he? Why???

**Comment by misutoneko** [comment]
Most likely, that's the person who did the translation (it comes from the training data).
The model thinks it belongs there because it was trained with similar audio where the text was present.
If you have any samples to share you can post them [here](https://github.com/openai/whisper/discussions/1873).

**Comment by unitythemaker** [comment]
> Most likely, that's the person who did the translation (it comes from the training data).
> The model thinks it belongs there because it was trained with similar audio where the text was present.
> If you have any samples to share you can post them [here](https://github.com/openai/whisper/discussions/1873).

Yeah, we have the same thing for the Turkish version of Whisper. It keeps saying Altyazı M.K. for the silent parts which means like Subtitles by M.K.
It is very annoying.

**Comment by kerdl** [comment]
DimaTorzok originated from [these](https://filmot.com/search/DimaTorzok/1?searchManualSubs=1&startDate=2005-01-01&endDate=2022-09-21&sortField=uploaddate&sortOrder=asc&gridView=1&)

Whoever trained the model forgot to sanitize, now it associates outro noises with this caption

**Comment by inoyakaigor** [comment]
Met this guy in subtitles for videos in Twitter

**Comment by KarelWintersky** [comment]
same issue with  `large-v3` model...

**Comment by Tureckiy-zart** [comment]
I hate this fucking Dima!

**Comment by ShirakamiSan** [comment]
Dima wtf?

**Comment by masterrr** [comment]
Telegram also has this problem haha

**Reply by W25X80** [reply]
it's very annoying

**Comment by CatRinaFaber** [comment]
Just imagine "DimaTorzok study case" in future ebooks
